internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -609,6 +609,18 @@ func convertEntryPoint(entrypoint manifest.EntryPointOverride) ([]string, error)
 	return out, nil
 }
 
+func getDeployConfig(deployStrategyName string) template.DeploymentConfigurationOpts {
+	var deployConfigs template.DeploymentConfigurationOpts
+	if deployStrategyName == "recreate" {
```

Comment:
let's use `strings.EqualFold` instead to make the manifest value case insensitive